src/ext/framework/mod.rs

```diff
@@ -258,6 +258,48 @@ impl Framework {
         self
     }
 
+    /// Aliases a command, allowing it to be used under a different name.
+    /// Note that default group is called "Ungrouped".
```

Comment:
`Note that the default group is named "Ungrouped".`

Empty doc line after the subject line please:
```rs
/// Aliases ...
///
/// Note that ...
```